.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```

Comment:
It doesn't look like your gitignore worked as expected, as there are 2030 files added to github. 

I believe changing /node_modules to node_modules should do the trick!